packages/components/tag/src/tag.ts

```diff
@@ -104,27 +104,35 @@ export class Tag extends ScopedElementsMixin(LitElement) {
     if (changes.has('disabled')) {
       this.setAttribute('tabindex', this.disabled ? '-1' : '0');
     }
+
+    if (changes.has('removable')) {
+      if (this.removable) {
+        this.setAttribute('aria-description', msg('Press the delete or backspace key to remove this tag'));
+      } else {
+        this.removeAttribute('aria-description');
+      }
+    }
   }
 
   override render(): TemplateResult {
     return html`
-      <div class="wrapper" .tabIndex=${this.disabled ? -1 : 0}>
```

Comment:
So currently we cannot focus on click the whole tag as we wanted before, is it ok?

Reply:
And I think we wanted before to remove the tag only when the remove button was focused (with keyboard), not when the whole tag gets focus? I think the case was also that we could make selectable tags in the future?

Reply:
As discussed in yesterday's meeting: have the ability to use the keyboard to delete a tag improves the UX for people using the keyboard/screen readers.